### PR TITLE
commit

### PR DESCRIPTION
**Pull request by lawebdeprogramador**
If you are submitting a new blog please read:

By sending this pull request to add my blog I verify that I adhere to the Planet Xamarin blog guidelines (please check each item that applies):

- [ yes] I have a valid blog & RSS URL
- [ yes] I host NO malicious or offensive content on the blog (including photos, swearing, etc.)
- [ yes] My blog is active with at least 3 Xamarin related blog posts in the last 6 months
- [ yes] I have applied a filter (Optional: if blog is only about Xamarin)
- [ yes] I understand, if I delete my blog, I will be deleted from Planet Xamarin
- [ yes] I understand, my blog may be removed at any time if any of these guidelines are broken.

### CODE REVIEW

**Comment by lawebdeprogramador** [comment]
ok

**Comment by Cheesebaron** [comment]
Closing this PR.

1. you are not filling out the pull request template correctly
2. you are making way too many commits compared to what you are trying to achieve
3. your feed url is _still_ wrong
4. your blog supports categories, however you are not using them
5. you write about content covered by many other people including the official Xamarin documentation
6. code snippets are not formatted, making your blog hard to follow
7. you've prematurely added the Planet Xamarin featured blog image, even though your blog has not been accepted yet
8. the images you use in a lot of your blog posts are taken directly from the Xamarin documentation without naming the source or quoting it.
